src/ResourceManager/HDInsight/Commands.HDInsight.Test/ScenarioTests/DataLakeStoreScenarioTests.cs

```diff
@@ -10,11 +10,18 @@ public DataLakeStoreScenarioTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact]
+        [Fact (Skip = "Test requires fixing GraphAPI by upgrading to latest test framework")]
```

Comment:
@markcowl I had upgraded the graph rbac management client to 3.2.0-preview. It still didn't work.
I dug around a bit more and I figured that the new Graph Rbac mgmt client is auto-rest based...and HDInsight was trying to instantiate it using TestBase class from Microsoft.Azure.Test as opposed to Microsoft.Rest.ClientRuntime.Azure.TestFramework.

I tried to make the changes to instantiate HDInsightManagementClient by using Microsoft.Azure.Test ; and GraphRbacManagementClient by Microsoft.Rest.ClientRuntime.Azure.TestFramework....but I ran into some issues.

The call create a graphrbacmgmtclient -- context.GetGraphServiceClient<GraphRbacManagementClient>(environment); requires an argument of type TestEnvironment.
The call to create an instance of TestEnvironment -- TestEnvironmentFactory.GetTestEnvironment(); -- fails with an exception "System.InvalidOperationException -- The current SynchronizationContext may not be used as a TaskScheduler".

I am not yet sure why I am running into this issue.